_posts/2018-07-12-coworking-hardware.markdown

```diff
@@ -0,0 +1,9 @@
+---
+title: Coworking, theme: Programming Gadgets
```

Comment:
the colon `theme:` will cause a problem with the YAML header, better to use a dash